core/src/Revolution/Mail/modPHPMailer.php

```diff
@@ -221,8 +221,10 @@ public function send(array $attributes = [])
         try {
             if (strpos($this->mailer->ContentType, 'html') !== false) {
                 if (!empty($this->mailer->Body)) {
-                    $emogrifier = new Emogrifier($this->mailer->Body);
-                    $this->mailer->Body = $emogrifier->emogrify();
+                    $html = new InlineStyle($this->mailer->Body);
+                    /** @noinspection PhpParamsInspection */
```

Comment:
What is the purpose of this @noinspection tag?

Reply:
Would be useful to have this information in the code as well. 😄

Reply:
Interestingly, that invalid phpdoc is fixed in master branch since May 2018 (see https://github.com/christiaan/InlineStyle/commit/e7dbf8830507d63c032285ae34b2d4e947f99515). That gives me some concern, though not that much if the thing works. I'll see if I can't get someone to create a new release.

Reply:
Going to move forward with this for now though…